possible-interfaces.lisp

```diff
@@ -0,0 +1,27 @@
+;; select: selects a subset of columns from a dataframe.
+(select df col1 col2 col3)
+
+;; filter: filters rows of a dataframe to those that satisfy some conditions.
+(filter df (< col1 col2))
+
+(filter df (and (< col1 col2)
```

Comment:
This looks cool, I see only one obstacle. Right now column names aren't symbols. For example, in my project, I have column names like `"Prof 0.25"`.

I see a few ways to overcome this obstacle:

1) Place a restriction on column names to be only keywords.
2) Separate a notion of the column identifier and column name. This way column identifier should be a keyword whereas column name should be a string. We might make a column name optional and generate it from the column identifier by default.

I'd prefer the second way, because it allows to render dataframes in a more human readable way.

Reply:
I agree that the second way sounds better.